assets/app/view/tiles_page.rb

```diff
@@ -39,7 +39,7 @@ def render
         else
           # apparently separating rotations in URL with '+' works by passing ' '
           # to split here
```

Comment:
pretty sure this comment was originally mine, doesn't really make sense with `split` taking no arg now
```suggestion
```